query_server/test/cases/ddl/describe_database.sql

```diff
@@ -0,0 +1,15 @@
+DROP DATABASE IF EXISTS test1;
+
+DESCRIBE DATABASE test1;
+
+CREATE DATABASE IF NOT EXISTS test1;
+
+DESCRIBE DATABASE test1;
+
+CREATE DATABASE IF NOT EXISTS describetest2;
+
+DESCRIBE DATABASE describetest2;
+
```

Comment:
you can add a line
```
--#SORT=true
```
to sort the result set
You can see query_server/test/README_CN.md

Reply:
no effect